### PR TITLE
chore(refactor): clean new-ui temporary variable

### PR DESCRIPTION
**Pull request by ifaouibadi**

### CODE REVIEW

File: src/popup/router/components/NodeConnectionStatus.vue [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default {
   bottom: env(safe-area-inset-bottom);
   left: 0;
   right: 0;
-  background: variables.$color-bg-3-new;
+  background: variables.$color-bg-3;
```
Comment (peronczyk) [inline src/popup/router/components/NodeConnectionStatus.vue:52]: If we want to be sure that the message has the same bg color as the page I suggest to use CSS variable: `--screen-bg-color`.

File: src/popup/router/components/Plate.vue [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
   &.only-top,
   .top {
-    background-color: variables.$color-bg-3-new;
+    background-color: variables.$color-bg-3;
```
Comment (peronczyk) [inline src/popup/router/components/Plate.vue:29]: Same here, I guess the Plate should inherit the color from current screen with the use of `--screen-bg-color`.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/router/pages/AccountDetails.vue [hunk 3]
```diff
@@ -222,7 +222,7 @@ export default {
     gap: 16px;
     padding: 4px;
     border-radius: 14px;
-    background-color: variables.$color-bg-3;
+    background-color: var(--screen-bg-color);
```
Comment (peronczyk) [inline src/popup/router/pages/AccountDetails.vue:225]: This is actually wrong, because the tabs background should be darker than the screen background.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/router/pages/AccountDetails.vue (hunk 3 above)
Comment (ifaouibadi) [reply src/popup/router/pages/AccountDetails.vue:225]: i only replaced the backgrounds that have the same variables bg-3

**Review by pstachel — APPROVED** [review]

**Review by paolomolo — APPROVED** [review]

**Pull request closed without merging**